docker-compose.dev.yml

```diff
@@ -3,12 +3,17 @@ version: '3'
 services:
   meican:
     image: meican-app-dev
+    env_file:
+      - .env
+    entrypoint: ["./docker_for_build/wait-for-mysql.sh", "db", "/docker-entrypoint.sh"]
     build:
       context: ./docker_for_build
       dockerfile: Dockerfile_dev
       args:
         - MEICAN_VERSION
     container_name: meican-app-dev
+    links:
+      - db
```

Comment:
I believe it doesn't need that part as it already exists in the docker-compose.yml file

Reply:
Thanks for the suggestion, @leonardolauryel. You are right, and there is no need to define it here again since we are just extending the compose file from the `docker-compose.yml`. I've removed it on commit 4f6684cd